runtests.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
Run this command as listed below. If a server is already running it will not start a new one.

```bash
./runtests.sh                 # run with a small single node cluster
./runtests.sh --cluster   # run with a small 2 node cluster
```